cpp/ps0001/p001-soln.cpp

```diff
@@ -1,15 +1,19 @@
 #include <iostream> 
 using namespace std;
 int main() 
-{ int number_one;
+{
+  // prompting user for 3 numbers to add 1/2
+  short number_one;
   cin>>number_one;
-  int number_two;
+  short number_two;
   cin>>number_two;
-  int number_three;
+  short number_three;
   cin>>number_three;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add